docs/date.md

```diff
@@ -646,17 +646,13 @@ This method overrides `Object.prototype.valueOf()` and always throws an exceptio
 This is because it's not possible to compare `Temporal.PlainDate` objects with the relational operators `<`, `<=`, `>`, or `>=`.
 Use `Temporal.PlainDate.compare()` for this, or `date.equals()` for equality.
 
-### date.**toZonedDateTime**(_timeZone_: _timeZone_ : object | string, _time_?: Temporal.PlainTime | object | string, _options_?: object) : Temporal.ZonedDateTime
+### date.**toZonedDateTime**(_item_: object) : Temporal.ZonedDateTime
 
 **Parameters:**
 
-- `timeZone` (optional string or object): The time zone in which to interpret `date` and `time`, as a `Temporal.TimeZone` object, an object implementing the [time zone protocol](./timezone.md#protocol), or a string.
-- `time` (optional `Temporal.PlainTime` or value convertible to one): A time of day on `date`.
-- `options` (optional object): An object with properties representing options for the operation.
-  The following options are recognized:
-  - `disambiguation` (string): How to disambiguate if the date and time given by `date` and `time` does not exist in the time zone, or exists more than once.
-    Allowed values are `'compatible'`, `'earlier'`, `'later'`, and `'reject'`.
-    The default is `'compatible'`.
+- `item` (object): an object with properties to be added to `time`. The following properties are recognized:
+  - `time` (optional `Temporal.PlainTime` or value convertible to one): a time of day on `date` used to merge into a `Temporal.ZonedDateTime`.
+  - `timeZone` (required `Temporal.TimeZone` or value convertible to one, or an object implementing the [time zone protocol](./timezone.md#protocol)): the time zone in which to interpret `date` and `time`.
```

Comment:
```suggestion
  - `timeZone` (required `Temporal.TimeZone`, a value convertible to one like `'Asia/Tokyo'`, or an object implementing the [time zone protocol](./timezone.md#protocol)): the time zone in which to interpret `date` and `time`.
```

Here's another idea for editing this content. It's an easier edit than rewriting the sentence again, and probably more technically accurate than my previous edit in the Instant PR. ;-)